nntrainer/tensor/cache_loader.h

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2022 Jiho Chu <jiho.chu@samsung.com>
+ *
+ * @file   cache_loader.h
+ * @date   10 Nov 2022
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jiho Chu <jiho.chu@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  Cache loader class
+ *
```

Comment:
It might be better to understand, how it cooperate with task executed & task.
so.. ```CacheLoader``` initialized with ```TaskExecutoer``` which has ```Task``` maps and ```Task``` defines work (functional with atomic running) and type and priority.

Reply:
Maybe I'm misunderstang, but task executor already has task queue...

Reply:
yes. What I'm asking is more details about their relation.